### PR TITLE
feat(variable-table): apply variable type filter

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description

Apply variable type filter in variables table.

https://user-images.githubusercontent.com/83635051/210743476-fc067d9f-98b4-48f6-89d8-3cd143f5adec.mov

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/modules/DashboardManageVariableTable.vue [hunk 1]
```diff
@@ -92,17 +92,22 @@ const props = defineProps<Props>();
...
+                ...props.variables[d],
+                propertyName: d,
+            };
+            if (currentVariable.variable_type === 'MANAGED') return currentVariable;
+            return {
+                ...currentVariable,
+                managable: d,
+            };
+        });
+        if (state.selectedVariableType === 'ALL') return convertedVariables;
```
Comment (minseolee) [inline src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/modules/DashboardManageVariableTable.vue:107]: Please consider with making constant about `'MANAGED'` and `'ALL'` if they are used by other files!

**Review by minseolee — APPROVED** [review]
LGTM ~! Left a comment

File: src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/modules/DashboardManageVariableTable.vue (hunk 1 above)
Comment (piggggggggy) [reply src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/modules/DashboardManageVariableTable.vue:107]: I'd like to have some discussions about it next time!

**Pull request merged**